### DISCUSSION TITLE
I am trying to get headControls to work with the face filter. How can I do that?

### DISCUSSION THREAD

**Discussion by kmahfouz** [jeeliz/jeelizFaceFilter#286] [Q&A]
So, I am trying to make a sphere with a persons face in the center. When the person looks around they can see their face looking around as well, and the sphere rotates using headControls. Is this possible?

**Comment by xavierjs** [comment]
yes it is

**Reply by kmahfouz** [reply]
Do you have any idea how? I've been trying for a while. I tried doing the same thing that is done in the matrix example where they use threeInstances.faceObject.add() but that runs into the issue of not being able to init JeelizThreeHelper because you can't pass in spec. Thank you!

**Reply by kmahfouz** [reply]
@xavierjs

**Comment by xavierjs** [comment]
Hi

I charge for custom Three.js development services like this work. You can contact me at xavier_at_jeeliz.com for more information
